spring-cloud-sleuth-brave/src/main/java/org/springframework/cloud/sleuth/brave/autoconfig/TraceBraveAutoConfiguration.java

```diff
@@ -83,84 +82,78 @@
 	 */
 	public static final String DEFAULT_SERVICE_NAME = "default";
```

Comment:
Are these 3 constants brave specific? They don't seem to. If they don't, it feels odd to have public constants in an implementation auto-config.